cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -37,6 +37,10 @@ spec:
   skipUpgradeChecks: false
   # Whether or not continue if PGs are not clean during an upgrade
   continueUpgradeAfterChecksEvenIfNotHealthy: false
+  # WaitTimeoutForHealthyOSD defines the time (in minutes) the operator would wait before an OSD can be stopped for upgrade or restart.
```

Comment:
I only saw a const 1 minute being used for ok-to-stop. Did you intend to use this setting there as well?